package.json

```diff
@@ -13,8 +13,9 @@
     "@reduxjs/toolkit": "^1.9.7",
     "@web3-onboard/injected-wallets": "^2.10.8",
     "@web3-onboard/react": "^2.8.13",
-    "axios": "^1.6.0",
+    "axios": "^1.6.2",
```

Comment:
Axios is already installed. No need to update packages

Reply:
Revert the changes in package.json, delete yarn.lock and package-lock.json and node_modules folder and install reinstall ```npm install```